pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -1,4 +1,6 @@
 { lib, stdenv
```

Comment:
```suggestion
{ lib
, stdenv
```

Reply:
Seems unrelated to me.

Reply:
Marking this as resolved. We have not touched this part of code, so we think it should not be part of this PR. We are happy to resolve it in a following PR.